src/Commands/Seed.php

```
@@ -16,11 +16,22 @@ class Seed extends SeedCommand
     {
+        // See https://github.com/archtechx/tenancy/issues/1474
+        if (version_compare(app()->version(), '13.24.0', '>=')) {
+            $this->signature = 'tenants:seed
+                    {class? : The class name of the root seeder}
+                    {--class=Database\\Seeders\\DatabaseSeeder : The class name of the root seeder}
+                    {--database= : The database connection to seed}
+                    {--force : Force the operation to run when in production}';
+            $this->specifyParameters();
+        } else {
+            $this->name = 'tenants:seed';
+        }
```

PHPMD (2.15.0)

[error] The method __construct uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)